json/zitadel.json

```
@@ -0,0 +1,43 @@
+{
+    "name": "Zitadel",
+    "slug": "Zitadel",
+    "categories": [
+        0
```

I think catergory Authentication & Security fits better then Miscellaneous
```suggestion
        6
```